### ISSUE TITLE
tidy-pom plugin give false positive

### ISSUE THREAD

**Issue by romani** [checkstyle/checkstyle#821]
mvn clean site

cause:
```
[INFO] Generating "Linkcheck" report            --- maven-linkcheck-plugin:1.2:linkcheck
[INFO] Invoking the maven-site-plugin to ensure that all files are generated...
[ERROR] Error when invoking Maven, consult the invoker log file: /home/rivanov/checkstyle-test/checkstyle/target/invoker-site-plugin1337754815.txt
[ INFO] Begin to check links in files...
[ INFO] Links checked.
[INFO] Rendering 36 generated Doxia documents: 36 xdoc
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
```

in a referenced file:
```
Caused by: org.apache.maven.plugin.MojoFailureException: The POM violates the code style. Please format it by running `mvn tidy:pom`.
	at org.codehaus.mojo.tidy.CheckMojo.execute(CheckMojo.java:58)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:106)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	... 29 more
```

**Comment by romani** [comment]
@mkordas , please take a look, we need to resolve that till 6.5 release.

**Comment by romani** [comment]
fixed, will be in 6.5
